templates/invoices_app/invoice-list.html

```diff
@@ -8,33 +8,44 @@
 {% else %}
 {% include 'supplier_app/navbar.html' %}
 {% endif %}
-    <br/>
-    <h1 class="text-center"> {% trans "Invoice List"%}</h1>
-    <br/>
+    <div class="row">
+        <div class="col">
+        </br>
```

Comment:
What do you want to achieve with this two <br>? Just for spacing?

Reply:
yes, should I use margin in the row?

Reply:
One approach could be add this around the h1: `<div class="pt-x pb-x`. You can use 1, 2, 3 or 4 in order to increase the padding top and bottom.